### ISSUE TITLE
At installaiton ImportError: cannot import name 'create_toprow' from 'modules.ui' Automatic1111 version: 1.6.0

### ISSUE THREAD

**Issue by StephZarx** [CiaraStrawberry/TemporalKit#97]
Hello,

*** Error loading script: sd-TemporalKit-UI.py
    Traceback (most recent call last):
      File "E:\AI\stable-diffusion-webui-1.6.0\modules\scripts.py", line 382, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "E:\AI\stable-diffusion-webui-1.6.0\modules\script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 940, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "E:\AI\stable-diffusion-webui-1.6.0\extensions\TemporalKit\scripts\sd-TemporalKit-UI.py", line 23, in <module>
        from modules.ui import create_toprow, create_sampler_and_steps_selection
    ImportError: cannot import name 'create_toprow' from 'modules.ui' (E:\AI\stable-diffusion-webui-1.6.0\modules\ui.py)

After tqdm error (solved by copy from system), this one appears with a fresh Automatic1111 v1.6 installation.

So installation is not possible and the tab extension does not appear.

An idea ?

**Comment by nurullahozatak** [comment]
I have same problem and didn't figure it out. I will switch older version of the  A1111

**Comment by hafedh-hichri** [comment]
the **reason** behind this is that A1111 changed the file under `modules/ui.py` by changing the `create_toprow` to `Toprow` .

luckily, in TemporalKit this is an extra import and it's not needed in the extension.
you can fix this by 
* open the file inside of your A1111 directory `extensions\TemporalKit\scripts\sd-TemporalKit-UI.py`
* change the following line    `from modules.ui import create_toprow, create_sampler_and_steps_selection` into 
``` 
from modules.ui import  create_sampler_and_steps_selection 
```

**Comment by UriahKong** [comment]
> 

thanks bro

**Comment by StephZarx** [comment]
This workaround is perfect. Thank you.

**Comment by darkflare** [comment]
> the **reason** behind this is that A1111 changed the file under `modules/ui.py` by changing the `create_toprow` to `Toprow` .
> 
> luckily, in TemporalKit this is an extra import and it's not needed in the extension. you can fix this by
> 
>     * open the file inside of your A1111 directory `extensions\TemporalKit\scripts\sd-TemporalKit-UI.py`
> 
>     * change the following line    `from modules.ui import create_toprow, create_sampler_and_steps_selection` into
> 
> 
> ```
> from modules.ui import  create_sampler_and_steps_selection 
> ```

Fixed! on 1.6.0 thanks mate!

**Comment by alexbofa** [comment]
Fix here https://github.com/CiaraStrawberry/TemporalKit/pull/91
